src/sentry/static/sentry/app/components/list.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import theme from 'app/utils/theme';
+import space from 'app/styles/space';
+
+export const List = styled('ul')`
+  list-style: none;
+  padding: 0;
+  margin-bottom: ${space(2)};
+
+  ol& {
+    counter-reset: numberedList;
+  }
+`;
+
+type ListItemProps = {
+  icon?: React.ReactNode;
+  children?: string | React.ReactNode;
+  className?: string;
+};
+
+const IconWrapper = styled('span')`
+  display: flex;
+  margin-right: ${space(1)};
+
+  /* Give the wrapper an explicit height so icons are line height with the
+   * (common) line height. */
+  height: 16px;
+  align-items: center;
+`;
+
+export const ListItem = styled(({icon, className, children}: ListItemProps) => (
+  <li className={className}>
+    {icon && <IconWrapper>{icon}</IconWrapper>}
+    {children}
+  </li>
+))<ListItemProps>`
+  display: flex;
+  align-items: center;
+  position: relative;
+  padding-left: 34px;
+  margin-bottom: ${space(0.5)};
+
+  &:before,
+  & > span:first-of-type {
```

Comment:
```suggestion
  & > ${IconWrapper} {
```

The new selector is safer than before, but could still cause problems if there is no icon and the children contains a span. I think this is what you're aiming to do and should have less splash damage.

Reply:
Gasp! This is so much better. Fixed.